### PR TITLE
Doc re-branding across all Zowe versions

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: ik673626 <igor.kazmyr@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
This is a rather big PR because it addresses the re-branding effort across all existing versions of Zowe docs. Please pay attention while reviewing the changes and leave your comments against lines in files that you find problematic. I will then edit those files.
Any questions, let me know in the comment section of this PR. Many thanks!

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/configure-certificates-keystore.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ To customize the `ZWESSOTK` JCL, edit the JCL variables at the beginning of the
 
 #### `PRODUCT` variable
 
-The `PRODUCT` variable specifies the z/OS security manager. The default value is `RACF`. Change the value to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or CA Top Secret for z/OS as your z/OS security manager.
+The `PRODUCT` variable specifies the z/OS security manager. The default value is `RACF`. Change the value to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or Top Secret for z/OS as your z/OS security manager.
```
Comment (janan07) [inline docs/user-guide/configure-certificates-keystore.md:174]: Review this reference with API ML squad. The reference Access Control Facility CA-ACF2 appears in the files configure-certificate-keystore and configure-certificates-keyring

Comment (janan07) [reply docs/user-guide/configure-certificates-keystore.md:174]: This reference can be replaced with ACF2 for z/OS

**Comment by JamesBauman** [comment]
HI all - I'm sorry for the delay :| 

Nonetheless, it appears that the ref's to CA were removed as required.  It's been a while for me, I am not sure what actions to perform to 'approve' the PR.

Also, does anyone know if my DCO is still active/in effect?

Thx,

File: docs/user-guide/configure-certificates-keyring.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ To customize the `ZWEKRING` JCL, edit the JCL variables at the beginning of the
 
 ### `PRODUCT` variable
 
-The `PRODUCT` variable specifies the z/OS security manager.  The default value is `RACF`. Change the value to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or CA Top Secret for z/OS as your z/OS security manager.  
+The `PRODUCT` variable specifies the z/OS security manager.  The default value is `RACF`. Change the value to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or Top Secret for z/OS as your z/OS security manager.
```
Comment (nannanli) [inline docs/user-guide/configure-certificates-keyring.md:28]: Should this `CA-ACF2` be rebranded as well?

**Review by nannanli — APPROVED** [review]
Just one minor comment. Otherwise, it looks good to me.

**Comment by nannanli** [comment]
@JamesBauman Hi Jim, the DCO has not changed so I think your DCO is still active.

File: docs/user-guide/configure-certificates-keystore.md (hunk 1 above)
Comment (IgorCATech) [reply docs/user-guide/configure-certificates-keystore.md]: Addressed with another commit

File: docs/user-guide/configure-certificates-keyring.md (hunk 2 above)
Comment (IgorCATech) [reply docs/user-guide/configure-certificates-keyring.md]: Thanks a lot! Addressed with another commit :)

**Comment by IgorCATech** [comment]
> HI all - I'm sorry for the delay :|
> 
> Nonetheless, it appears that the ref's to CA were removed as required. It's been a while for me, I am not sure what actions to perform to 'approve' the PR.
> 
> Also, does anyone know if my DCO is still active/in effect?
> 
> Thx,

Hey Jim, no worries. I've made sure to remove all the references across the existing versions of Zowe Docs in this PR. However, because there are quite a few of them, I asked you, Andrew, and Ashley to help me check if I didn't miss any reference to CA anywhere in the docs. If everything looks good to you, please approve the PR :)
Any questions, let me know!

**Review by IgorCATech — COMMENTED** [review]
I've just scanned all the docs, and have committed the change for "Access Control Facility CA-ACF2" across all versions of the doc.
Otherwise, the change looks good

**Review by janan07 — APPROVED** [review]
I think you caught all of the references.

**Pull request merged**